src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -260,3 +260,28 @@ def set_quotas(self, path, max_bytes=None, max_files=None):
         if max_files is not None:
             self.cfs.setxattr(path, 'ceph.quota.max_files',
                               str(max_files).encode(), 0)
+
+    def get_statfs(self, path):
+        """
+        Get the statfs of the specified path by xattr.
+        :param path: The path of the directory/file.
+        :type path: str
+        :return: Returns a dictionary containing 'used_bytes'
+            and 'used_files'.
+        :rtype: dict
+        """
+        logger.debug("get_statfs by xattr of directory: %s", path)
```

Comment:
is this debug line really needed?

Reply:
Not really needed, I will follow your suggestion here.